### PR TITLE
docs: init mkdocs and move documentation from DSP-DOCS into DSP-APP repo (DSP-380)

### PR DESCRIPTION
**Pull request by kilchenmann**
resolves DSP-380

I copied the user-guide from dsp-docs (docs.dasch.swiss) to this repo. I tried to clean up the structure in the main README. The big refactoring will be part of another PR. 
The DSP-APP github page is already published: <https://dasch-swiss.github.io/dsp-app/>

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+## Contribute
+
+If you would like to add your own contributions to the docs, please read the following information regarding the file structure to ensure you follow the same structure.
+
+### File structure
+
+The documentation consists of three main topics with subordinate themes:
+
+1. **index** contains all information about the DSP-APP
+1. **how-to-use** contains all about the usage of the DSP-APP modules
```
Comment (flavens) [inline docs/README.md:15]: "contains all about" > weird to me
do you mean "all the information about..."

**Review by flavens — APPROVED** [review]

File: docs/README.md (hunk 1 above)
Comment (kilchenmann) [reply docs/README.md:15]: yes it's weird... I'll fix it

Comment (flavens) [reply docs/README.md:15]: but as you said, "the big refactoring will be part of another PR." So you could aadd this point to the other PR.

Comment (kilchenmann) [reply docs/README.md]: > but as you said, "the big refactoring will be part of another PR." So you could aadd this point to the other PR.

sure... but the content came from DSP-UI documentation and has to be updated for DSP-APP. DSP-APP documentation has another structure than DSP-UI

resolved it in 4440fc4

**Review by flavens — APPROVED** [review]
the English should be corrected in the next PR

**Pull request merged**